app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = FoodService.get_recipes(params[:search])
+  end
+
+  private
+
+  def recipe_params
+    params.require(:recipe).permit(:search)
+  end
+end
```

Comment:
Does this not belong to another PR?

Reply:
No it's our landing-page.  :) We need to have it.